at_end2end_test/test/plookup_verb_test.dart

```diff
@@ -64,15 +68,26 @@ void main() {
     await sh1.writeCommand('plookup:no-key$atSign_1');
     String response = await sh1.read();
     print('plookup verb response $response');
-    expect(response, contains('error:AT0015-key not found : public:no-key$atSign_1 does not exist in keystore'));
+    if (atSign_1 != '@cicd5') {
```

Comment:
Please create a github issue to amend these tests once cicd5 is running server version 3.0.21 (i.e. once we have promoted current canary to all of prod)

Reply:
Alternatively, rewrite this test so the if condition is not about the atSign, but about the version of the server software that the atSign is running